packages/protocol/contracts-0.8/common/interfaces/ICeloToken.sol

```diff
@@ -1,34 +0,0 @@
-// SPDX-License-Identifier: UNLICENSED
```

Comment:
Was this deleted on purpose?

Reply:
yes, since the interface in `contracts/common/interfaces` is sufficient.